rundeckapp/grails-app/services/rundeck/services/ExecutionService.groovy

```diff
@@ -908,6 +908,7 @@ class ExecutionService implements ApplicationContextAware, StepExecutor, NodeSte
         jobcontext.loglevel = textLogLevels[execution.loglevel] ?: execution.loglevel
         jobcontext.retryAttempt=Integer.toString(execution.retryAttempt?:0)
         jobcontext.wasRetry=Boolean.toString(execution.retryAttempt?true:false)
+        jobcontext.threadcount=Integer.toString(execution.nodeThreadcount)
```

Comment:
execution.nodeThreadcount *might* be null, so perhaps it should add `?:1` to guard against that case

Reply:
Thanks, just made the change so it will default to 1 in that case.